File: src/FirstTracks.Core/Models/SkiResortSnowfall.cs
namespace FirstTracks.Core.Models
{
	public class SkiResortSnowfall : BaseModel
	{
		public string SkiResortSnowfallId { get; set; }
		public int Month { get; set; }
		public int Year { get; set; }
		public int Inches { get; set; }
	}
}

File: src/FirstTracks.Web/Scripts/Models/SkiResortSnowfall/SkiResortSnowfall.ts
export class SkiResortSnowfall {
    skiResortSnowfallId: string;
    month: number;
    year: number;
    inches: number;
    createdOn: Date;
    modifiedOn: Date;

    constructor(skiResortSnowfallId: string,
        month: number,
        year: number,
        inches: number,
        createdOn: Date,
        modifiedOn: Date)
    {
        this.skiResortSnowfallId = skiResortSnowfallId;
        this.month = month;
        this.year = year;
        this.inches = inches;
        this.createdOn = createdOn;
        this.modifiedOn = modifiedOn;
    }
}
export default SkiResortSnowfall;